de.nicokimmel.shadowcast-electron.json

```diff
@@ -0,0 +1,53 @@
+{
+  "app-id": "de.nicokimmel.shadowcast-electron",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "base": "org.electronjs.Electron2.BaseApp",
+  "base-version": "23.08",
+  "command": "run.sh",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--device=all",
+    "--socket=pulseaudio",
+    "--share=network"
+  ],
+  "modules": [
+    {
+      "name": "shadowcast-electron",
+      "buildsystem": "simple",
+      "build-options": {
+        "env": {
+          "XDG_CACHE_HOME": "/run/build/shadowcast-electron/flatpak-node/cache",
+          "npm_config_cache": "/run/build/shadowcast-electron/flatpak-node/npm-cache",
+          "npm_config_offline": "true"
+        }
+      },
+      "build-commands": [
+        "ls -la",
+        "mkdir /app/main",
+        "cp -r shadowcast-electron/* /app/main",
+        "install -Dm644 shadowcast-electron/resources/app/flatpak/de.nicokimmel.shadowcast-electron.desktop /app/share/applications/de.nicokimmel.shadowcast-electron.desktop",
+        "install -Dm644 shadowcast-electron/resources/app/flatpak/de.nicokimmel.shadowcast-electron.metainfo.xml /app/share/metainfo/de.nicokimmel.shadowcast-electron.metainfo.xml",
+        "install -Dm644 shadowcast-electron/resources/app/src/public/images/icon.png /app/share/icons/hicolor/512x512/apps/de.nicokimmel.shadowcast-electron.png",
+        "install -Dm755 run.sh /app/bin/run.sh"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/nicokimmel/shadowcast-electron/releases/download/v1.0.0/shadowcast-electron-linux-x64-1.0.0.zip",
```

Comment:
how does that work on aarch64?

Reply:
Does it have to work on ARM?

Reply:
an aarch64 package is built. how does it work if you put x86_64 binaries?

Reply:
I dont build arm versions currently. But I could do so. But could you please tell me how to tell the flatpak builder which archives are for which arch?